presto/overlays/dev/presto-cr.yaml

```diff
@@ -0,0 +1,116 @@
+apiVersion: starburstdata.com/v1
+kind: Presto
+metadata:
+  name: dev-presto
+spec:
+  nameOverride: "dev-presto"
+  clusterDomain: cluster.local
+  environment: ""
+  additionalJvmConfigProperties: ""
+  additionalCatalogs: {}
+  additionalEtcPrestoTextFiles: {}
+  additionalEtcPrestoBinaryFiles: {}
+  licenseSecretName: ""
+  imageNamePrefix: ""
+  additionalBootstrapScriptVolume: {}
+  additionalBootstrapScriptVolumes: {}
+  additionalVolumes: []
+
+
+  prometheus:
+    additionalRules:
+    - name: failed_tasks
+      pattern: presto.execution<name=TaskManager><>FailedTasks.TotalCount
+      type: COUNTER
+    enabled: true
+
+  service:
+    name: presto-service
+    type: ClusterIP
+
+
+  image:
+    name: starburstdata/presto:338-e.3-k8s-0.41
```

Comment:
please move to quay?

Reply:
Good point, but the idea behind this PR is to have a dev overlay with the PostgreSQL JDBC URL configured for that environment. I can work on another PR to fix this.